src/utils/format.ts

```diff
@@ -206,7 +206,7 @@ export const isPercentageFormatValid = (value: string) => {
 };
 
 // Some locales have a comma decimal separator (e.g., "1.234,56" in German is "1,234.56" in USD).
-// This function normalizes a given currency or percentage for APIs.
+// This function normalizes a given currency or percentage for APIs where USD decimal format is expected.
 export const unFormat = (value: string) => {
```

Comment:
i wonder if the name of this should be changed to be more descriptive rather than having to add comments wherever it is being used.. maybe `normalizeToUsdDecimalFormat` even

Reply:
Sorry I missed this. Currency support is a work in progress, and will consider your suggestion for the next PR. For now, I'd like to keep the comments.

Reply:
Wanted to follow up on this. I gave the function name some thought, but would like to keep it and add comments to better describe how it's used. I forgot it's not only used for APIs.

1. Some locales us a comma as the decimal separator (e.g., "1.234,56" in German), which must be replaced for APIs where USD decimal format is expected.

2. The group separator (e.g., "1,234.56" in USD) must also be removed when formatting currencies and percentages to display in the browser's locale.